File: Domain/Route.cs
using System.Collections.Generic;

namespace Domain
{
    public class Route
    {
        public int Id { get; set; }
        public Node StartNode { get; set; }
        public Node EndNode { get; set; }
        public Transport Transport { get; set; }
        public int Duration { get; set; }
        public int VanguardTime { get; set; }
        public int Cost { get; set; }
        public List<DepartureDayTime> DepartureDayTime { get; set; } = new List<DepartureDayTime>();
    }
}

File: pli40app/models/route.js
class Route {
    constructor (id, startPoint, endPoint, transport, duration, cost, stops) {
        this.id = id;
        this.startPoint = startPoint;
        this.endPoint = endPoint;
        this.transport = transport;
        this.duration = duration;
        this.cost = cost;
        this.stops = stops;
    }
};

export default Route;